helpers/setURLQueryParam.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+const URL = require('url').URL;
+const utils = require('handlebars-utils');
+const common = require('./lib/common.js');
+const SafeString = require('handlebars').SafeString;
+
+const factory = () => {
+    return function(string, key, value) {
+        if (string instanceof SafeString) {
```

Comment:
I wonder if we should do similar checks for key and value ?

Reply:
makes sense, I'm updating